docker-for-aws/faqs.md

```diff
@@ -4,11 +4,29 @@ keywords: aws faqs
 title: Docker for AWS Frequently asked questions (FAQ)
 ---
 
-## Can I use my own AMI?
+### Stable and beta channels
```

Comment:
Please make sure you start with level-2 headings. If you skip a step, Jekyll can get very confused.